test/Test/Database/LSMTree/Internal/Snapshot/Codec.hs

```diff
@@ -152,6 +150,7 @@ testAll test = [
       test (Proxy @SnapshotMetaData)
     , test (Proxy @SnapshotLabel)
     , test (Proxy @SnapshotTableType)
+    , test (Proxy @SnapshotRun)
```

Comment:
Put it under `-- SnapLevels`

Reply:
It's not specific to levels, it's used in both levels and unions. I put it at the top because it's the type used to instantiate the `r` param everywhere within the `SnapshotMetaData`. So I think it does actually belong at the top here.